clients/vscode-hlasmplugin/package.json

```diff
@@ -20,7 +20,9 @@
     "z/OS",
     "zos",
     "zowe",
-    "ibm assembler"
+    "ibm assembler",
+    "language",
```

Comment:
language? That seems to be a bit "too universal", doesn't it?

Reply:
please feel free to suggest an alternative - something that would likely be written in the search bar on the markeplace

Reply:
well, I would have thought that 9 keywords would be ok... rather than adding 10th that's basically not saying anything.